core/msd/src/main/rdl/TransportPolicyRule.tdl

```diff
@@ -73,4 +87,15 @@ type TransportPolicyEgressRule Struct {
 type TransportPolicyRules Struct {
     Array<TransportPolicyIngressRule> ingress; // List of ingress rules
     Array<TransportPolicyEgressRule> egress; // List of egress rules
-}
\ No newline at end of file
+}
+
+type TransportPolicyValidateRule Struct {
+    Array<TransportPolicyPeer> source;
+    Array<TransportPolicyPeer> destination;
+    Array<String> instances (optional);
```

Comment:
What is the significance of this field?

Reply:
the instances field would allow the validation to be done only on the ips associated with the hosts mentioned instead of all the hosts with the service identity.